content/en/docs/setup/upgrade/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Upgrade Istio
 linktitle: Upgrade
-description: Upgrade, downgrade, and manage Istio accross multiple control plane revisions.
+description: Upgrade, downgrade, and manage Istio across multiple control plane revisions.
```

Comment:
This is the only change that is in this repo, so this PR will likely be this one file.